templates/admin/form.html.twig

```diff
@@ -0,0 +1,133 @@
+{# @see https://symfony.com/bundles/EasyAdminBundle/current/design.html#form-field-templates #}
+{% block _Theme_themeCategories_widget %}
+    {% set id = 'form-theme-themeCategories' %}
+    <table class="table table-striped table-ea-collection" id="{{ id }}">
+      <thead>
+        <th>{{ 'theme.form.sort_order'|trans }}</th>
```

Comment:
```suggestion
        <th scope="col">{{ 'theme.form.sort_order'|trans }}</th>
```

Reply:
https://webaim.org/techniques/tables/data

"The scope attribute tells the browser and screen reader that everything within a column that is associated to the header with scope="col" in that column, and that a cell with scope="row" is a header for all cells in that row."

Reply:
I don't exactly remember how this table looks, but scope gives a "header" to the cells, so if there is a button without a header in the last column that might be confusing for assistive tech.

Reply:
`scope` attributes added in https://github.com/itk-dev/sysstatus/pull/33/commits/fe975a024992f5dd268dbf93609da7742fca0624.